extensions/kudo/model/src/main/java/io/fabric8/kudo/api/model/KudoResourceMappingProvider.java

```diff
@@ -0,0 +1,58 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.kudo.api.model;
+
+import io.fabric8.kubernetes.api.KubernetesResourceMappingProvider;
+import io.fabric8.kubernetes.api.model.KubernetesResource;
+import java.util.HashMap;
+import java.util.Map;
+
+public class KudoResourceMappingProvider implements KubernetesResourceMappingProvider {
+
+    public final Map<String, Class<? extends KubernetesResource>> mappings = new HashMap<>();
+
+    public KudoResourceMappingProvider () {
+        /*mappings.put("serving.knative.dev/v1alpha1#Configuration", io.fabric8.knative.serving.v1alpha1.Configuration.class);
+        mappings.put("serving.knative.dev/v1alpha1#Revision", io.fabric8.knative.serving.v1alpha1.Revision.class);
+        mappings.put("serving.knative.dev/v1alpha1#Service", io.fabric8.knative.serving.v1alpha1.Service.class);
+        mappings.put("serving.knative.dev/v1alpha1#Route", io.fabric8.knative.serving.v1alpha1.Route.class);
+
+        mappings.put("serving.knative.dev/v1beta1#Configuration", io.fabric8.knative.serving.v1beta1.Configuration.class);
+        mappings.put("serving.knative.dev/v1beta1#Revision", io.fabric8.knative.serving.v1beta1.Revision.class);
+        mappings.put("serving.knative.dev/v1beta1#Service", io.fabric8.knative.serving.v1beta1.Service.class);
+        mappings.put("serving.knative.dev/v1beta1#Route", io.fabric8.knative.serving.v1beta1.Route.class);
+
+        mappings.put("serving.knative.dev/v1#Configuration", io.fabric8.knative.serving.v1.Configuration.class);
+        mappings.put("serving.knative.dev/v1#Revision", io.fabric8.knative.serving.v1.Revision.class);
+        mappings.put("serving.knative.dev/v1#Service", io.fabric8.knative.serving.v1.Service.class);
+        mappings.put("serving.knative.dev/v1#Route", io.fabric8.knative.serving.v1.Route.class);
+
+        mappings.put("eventing.knative.dev/v1alpha1#Broker", io.fabric8.knative.eventing.v1alpha1.Broker.class);
+        mappings.put("eventing.knative.dev/v1alpha1#Channel", io.fabric8.knative.messaging.v1alpha1.Channel.class);
+        mappings.put("eventing.knative.dev/v1alpha1#EventType", io.fabric8.knative.eventing.v1alpha1.EventType.class);
+        mappings.put("eventing.knative.dev/v1alpha1#Trigger", io.fabric8.knative.eventing.v1alpha1.Trigger.class);
+        mappings.put("eventing.knative.dev/v1alpha1#Subscription", io.fabric8.knative.messaging.v1alpha1.Subscription.class);
+
+        mappings.put("messaging.knative.dev/v1alpha1#InMemoryChannel", io.fabric8.knative.messaging.v1alpha1.InMemoryChannel.class);
+        mappings.put("messaging.knative.dev/v1alpha1#Sequence", io.fabric8.knative.flows.v1alpha1.Sequence.class);*/
+      mappings.put("kudo.dev/v1beta1#InstanceList", io.fabric8.kudo.api.model.v1beta1.InstanceList.class);
```

Comment:
I think that we have more classes we need to add.

Reply:
hi, added and commit the code.